web/app/components/header/account-setting/model-provider-page/declarations.ts

```diff
@@ -102,6 +102,7 @@ export type FormShowOnObject = {
 
 export type CredentialFormSchemaBase = {
   variable: string
+  inputs?: Array<{ name: string; value: string }>
```

Comment:
to render a form with an input that accepts variables, the schema is changed to accept `inputs` property, `variables` might be a better name, but it is confusing next to `variable`.